Simplify 2i (#60)
thewca/wca-regulations-january-2026: wca-regulations.md

    - 2g3) Competitors in a group in progress must not communicate with other competitors in the group about the scrambled states of the group, unless all such competitors are not eligible for any further attempts in the group. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves and 3x3x3 Multi-Blind, competitors that have finished their attempt may share and discuss the scramble sequence(s) with other competitors in the group who have either finished their attempt or declined their attempt.
        - 2g3a) For rounds where results are being entered live, the organization team may enforce that competitors are not permitted to view the results entered from their group until they have completed their attempts.
- 2h) Competitors must be appropriately dressed while in the competition venue. At the discretion of the WCA Delegate, competitors may be disqualified from the competition for inappropriate clothing.

> - 2i) While inspecting or solving, competitors may use aids that do not give an unfair advantage, at the discretion of the WCA Delegate. Penalty for using a non-permited aid: disqualification of the attempt (DNF). Exception: if a non-permitted aid is used, the current attempt and any completed attempts do not need be disqualified if the competitor did not gain an unfair advantage from the aid, at the discretion of the WCA Delegate, however the competitor must not be permitted to perform any further attempts with the aid.

Author: ```suggestion
- 2i) While inspecting or solving, competitors may use objects that do not give an unfair advantage, at the discretion of the WCA Delegate. Penalty for using a non-permited object: disqualification of the attempt (DNF). Exception: if a non-permitted object is used, the current attempt and any completed attempts do not need be disqualified if the competitor did not gain an unfair advantage from the object, at the discretion of the WCA Delegate, however the competitor must not be permitted to perform any further attempts with the object.
```

>     - 2i1) Examples of aids that should be permitted:

Author: ```suggestion
    - 2i1) Examples of objects that should be permitted:
```

        - 2i1a) Medical/physical aids worn by the competitor (e.g. glasses, wrist brace).

>         - 2i1b) Non-electronic earplugs and earmuffs, but not electronic headphones and earbuds (see [Regulation 2i3a](regulations:regulation:2i3a))

Author: ```suggestion
        - 2i1b) Non-electronic earplugs and earmuffs, but not electronic headphones and earbuds (see [Regulation 2i2a](regulations:regulation:2i2a))
```
Fix reference

        - 2i1c) Food and drink.
        - 2i1d) Electronic or non-electronic hand warmers.
        - 2i1e) Cameras, including cellphones used as cameras. Camera monitors must be blank or out of sight of the competitor.

>     - 2i2) The following aids must not be permitted to use during an attempt:

Author: ```suggestion
    - 2i2) The following objects must not be permitted to use during an attempt:
```

    - 3a1+) [CLARIFICATION] Competitors may be disqualified if they do not have a puzzle ready to submit soon after they are called (e.g. if they planned to use a puzzle that another competitor is currently using, and therefore cannot submit a puzzle at the moment).
    - 3a2) Puzzles must be fully operational, such that normal scrambling is possible.
    - 3a3) Polyhedral puzzles must use a color scheme with one unique color per face in the solved state. Each puzzle variation must have moves, states, and solutions functionally identical to the original puzzle.
    - 3a4) Puzzles must not have electronic components (e.g. Bluetooth or Wi-Fi capabilities, motors, sensors, lights).

> - 3d) Puzzles must have colored parts, which define the color scheme of the puzzle and must be one and only one of the following: colored stickers, colored tiles, colored plastic, or painted/printed colors. All colored parts of a puzzle must be made of a similar material.

Author: ```suggestion
- 3d) Puzzles must have colored parts, which define the color scheme of the puzzle and must be one and only one of the following: colored stickers, colored tiles, colored plastic, or painted/printed colors. All colored parts of a puzzle must be made of a similar material. Exception: for Clock, puzzles do not have colored parts.
```

> - 3d) Puzzles must have colored parts, which define the color scheme of the puzzle and must be one and only one of the following: colored stickers, colored tiles, colored plastic, or painted/printed colors. All colored parts of a puzzle must be made of a similar material.

Author: ```suggestion
- 3d) Puzzles must have colored parts, which define the color scheme of the puzzle and must be one and only one of the following: colored stickers, colored tiles, colored plastic, or painted/printed colors. All colored parts of a puzzle must be made of a similar material. Exception: for Clock, puzzles do not have colored parts.
```

        - A1a4+) [REMINDER] If a competitor has accidentally been permitted to exceed the time limit, the time limit must be enforced retroactively, and the judge, competitor, and WCA Delegate should be informed (see [Regulation 1g2](regulations:regulation:1g2)). Judges must always be aware of the time limit for a current attempt (which might depend on previous attempts, in the case of a cumulative time limit).
        - A1a5) The time counting towards the time limit for an attempt is either the result after time penalties are applied (if the result was not DNF) or the time elapsed in the solve (if the result was DNF).
    - A1b) If the time limit for an attempt is greater than 10 minutes, a stopwatch must be used for timekeeping.

>     - A1b+) [ADDITION] The competitor may pick up a stopwatch to view the current time (when they are not blindfolded), they but not start, stop, pause, or otherwise interact with the timekeeping of the stopwatch. Exception: The competitor may stop the stopwatch at the end of their attempt if the judge is not immediately available (see [Regulation A6a1](regulations:regulation:A6a1)).

Reviewer: ```suggestion
    - A1b+) [ADDITION] The competitor may pick up a stopwatch to view the current time (when they are not blindfolded), they must not start, stop, pause, or otherwise interact with the timekeeping of the stopwatch. Exception: The competitor may stop the stopwatch at the end of their attempt if the judge is not immediately available (see [Regulation A6a1](regulations:regulation:A6a1)).
```

        - A1a5) The time counting towards the time limit for an attempt is either the result after time penalties are applied (if the result was not DNF) or the time elapsed in the solve (if the result was DNF).
    - A1b) If the time limit for an attempt is greater than 10 minutes, a stopwatch must be used for timekeeping.
    - A1b+) [ADDITION] The competitor may pick up a stopwatch to view the current time (when they are not blindfolded), they must not start, stop, pause, or otherwise interact with the timekeeping of the stopwatch. Exception: The competitor may stop the stopwatch at the end of their attempt if the judge is not immediately available (see [Regulation A6a1](regulations:regulation:A6a1)).
    - A1b++) [ADDITION] The organization team may provide the competitor an unofficial stopwatch for viewing the elapsed time (started together with the main stopwatch), in which case the competitor is not permitted to touch the official stopwatch. Exception: See the exception in [Regulation A6a1](regulations:regulation:A6a1).

>         - A1b1) A Stackmat must be used in addition to the stopwatch. Exception: for events without inspection where the competitor is expected to exceed 10 minutes, use of the Stackmat is optional.

Author: ```suggestion
        - A1b1) A speedsolving timer must be used in addition to the stopwatch. Exception: for events without inspection where the competitor is expected to exceed 10 minutes, use of the speedsolving timer is optional.
```

    - A1b) If the time limit for an attempt is greater than 10 minutes, a stopwatch must be used for timekeeping.
    - A1b+) [ADDITION] The competitor may pick up a stopwatch to view the current time (when they are not blindfolded), they must not start, stop, pause, or otherwise interact with the timekeeping of the stopwatch. Exception: The competitor may stop the stopwatch at the end of their attempt if the judge is not immediately available (see [Regulation A6a1](regulations:regulation:A6a1)).
    - A1b++) [ADDITION] The organization team may provide the competitor an unofficial stopwatch for viewing the elapsed time (started together with the main stopwatch), in which case the competitor is not permitted to touch the official stopwatch. Exception: See the exception in [Regulation A6a1](regulations:regulation:A6a1).
        - A1b1) A speedsolving timer must be used in addition to the stopwatch. Exception: for events without inspection where the competitor is expected to exceed 10 minutes, use of the speedsolving timer is optional.

>         - A1b2) If a time from the Stackmat timer is available, it is the original recorded time. Otherwise (e.g. the Stackmat timer malfunctioned, ran for over 10 minutes or was not used at all), the stopwatch time is the original recorded time.

Author: ```suggestion
        - A1b2) If a time from the speedsolving timer is available, it is the original recorded time. Otherwise (e.g. the speedsolving timer malfunctioned, ran for over 10 minutes or was not used at all), the stopwatch time is the original recorded time.
```

    - E3d) Stopwatch or watch (self-supplied) for keeping track of the elapsed time, if it is approved by the WCA Delegate.
    - E3d+) [CLARIFICATION] Any stopwatch or watch used by a competitor must not have any functions that would help the competitor find a solution.
    - E3d++) [CLARIFICATION] Competitors should not consider a personal stopwatch or watch as the official time, and must submit their solution when the judge calls "STOP".

>     - E3e) Other aids that do not give an unfair advantage, at the discretion of the WCA Delegate (see [Regulation 2i](regulations:regulation:2i)).

Author: ```suggestion
    - E3e) Other objects that do not give an unfair advantage, at the discretion of the WCA Delegate (see [Regulation 2i](regulations:regulation:2i)).
```
